hail/python/hail/backend/py4j_backend.py

```diff
@@ -2,20 +2,20 @@
 import abc
 import json
 
+import requests
 import py4j
 import py4j.java_gateway
 
 import hail
 from hail.expr import construct_expr
-from hail.ir import JavaIR, finalize_randomness
+from hail.ir import finalize_randomness, JavaIR
 from hail.ir.renderer import CSERenderer
 from hail.utils.java import FatalError, Env
-from hail.expr.blockmatrix_type import tblockmatrix
-from hail.expr.matrix_type import tmatrix
-from hail.expr.table_type import ttable
-from hail.expr.types import dtype
 
-from .backend import Backend, fatal_error_from_java_error_triplet
+from .backend import ActionTag, Backend, fatal_error_from_java_error_triplet
+
+import http.client
+http.client._MAXLINE = 2 ** 20
```

Comment:
what's this about?

Reply:
I left a comment explaining, and here's an [SO post](https://stackoverflow.com/questions/63157046/python-http-request-exception-linetoolong). I'm not really sure what to do here. It felt reasonable to send back timings in a response header instead of smushing it into the response body, as it is metadata, but I was left with a decision of how to raise the max header size and I went with something overly generous. AFAIK there is no actual limit in HTTP.

Reply:
I was going to say I was worried about proxies but I remember this entire connection is controlled by us. Yeah this seems fine.